spec/requests/blog_spec.rb

```diff
@@ -2,7 +2,7 @@
   include BlogSpecHelpers
 
   describe 'GET /blog/some-article' do
-    subject(:get_blog_feed) { get('/blog/some-article', headers:) }
+    subject(:get_blog_article) { get('/blog/some-article', headers:) }
```

Comment:
(Mostly unrelated, but I was looking at this spec since it started flaking in this PR. This subject was named incorrectly/badly/misleadingly (due to a lazy copy-paste, I think).)